experiments/golden-results/StratoX-summary.txt

```diff
@@ -18,17 +13,17 @@ Calling function: Process_Pragma_Declaration
 Error message: Unsupported pragma: Implementation defined
 Nkind: N_Pragma
 --
-Occurs: 107 times
-Calling function: Process_Declaration
-Error message: Generic declaration
-Nkind: N_Generic_Subprogram_Declaration
---
 Occurs: 99 times
 Calling function: Do_Function_Call
 Error message: func name not in symbol table
 Nkind: N_Function_Call
 --
-Occurs: 62 times
+Occurs: 99 times
```

Comment:
Am I right in thinking that these appear because they are the next error after the size issue?

Reply:
At least from previous experience, we just got a bit further in the AST and found some new unsupported nodes.